pkgs/development/python-modules/pymodbustcp/default.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "pyModbusTCP";
+  version = "0.3.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "sourceperl";
+    repo = "pyModbusTCP";
+    tag = "v${version}";
+    hash = "sha256-troGTvGTzOTsbux7QQ6L6sHvjPIU9VRLHqFOBHEHUl0=";
+  };
+
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "pyModbusTCP" ];
+
+  meta = {
+    description = "A simple Modbus/TCP client library for Python";
```

```suggestion
    description = "Modbus/TCP client library for Python";
```
Not start with the definite or an indefinite article.
Avoid subjective language.